dss/operations/params.py

```diff
@@ -0,0 +1,146 @@
+"""
+Get, set, and unset environment variables in the SSM store and in deployed lambda functions
+"""
+import boto3
+import os
+import typing
+import argparse
+import json
+import logging
+
+from dss.operations import dispatch
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_ssm_lambda_environment():
+    ssm_client = boto3.client("ssm")
```

Comment:
lets upgrade this to `dss.util.aws.clients`

Reply:
Addressed